utkdigitalinitiatives/TRACE
# Address TRAC-1028

** JIRA Ticket**: [TRAC-1028](link)
# What does this Pull Request do?
This PR updates the XML Form definition associated with islandora:compoundCModel.

# What's new?
I updated the XML Form definition that is stored in `configs/form-tables/xml_forms.sql`.

# How should this be tested?
Pull this feature branch and `vagrant destroy -f` and `vagrant up`. After the VM has finished booting, log in as `user[A|B] (or admin)`, and *submit* a Compound Object in  the `utk.ir:fg` collection. The form behavior should now allow for a correct submission.

# Additional Notes:
It looks like there may be an issue with editing a MODS datastream, post-submission, but that's a separate issue.

# Interested parties
@DonRichards @markpbaggett
